libraries/AP_HAL_ChibiOS/hwdef/scripts/defaults_normal.h

```diff
@@ -11,13 +11,18 @@
 #endif
 
 // a similar define is present in AP_HAL_Boards.h:
+// needed to compile chibios
 #ifndef HAL_OS_FATFS_IO
 #define HAL_OS_FATFS_IO 0
 #endif
 
+#ifndef HAL_OS_LITTLEFS_IO
+#define HAL_OS_LITTLEFS_IO 0
+#endif
+
 #ifndef AP_TERRAIN_AVAILABLE
 // enable terrain only if there's an SD card available:
-#define AP_TERRAIN_AVAILABLE HAL_OS_FATFS_IO
+#define AP_TERRAIN_AVAILABLE (HAL_OS_FATFS_IO || (HAL_OS_LITTLEFS_IO && (BOARD_FLASH_SIZE>1024)))
```

Comment:
Is LittleFS big enough for a meaningful amount? Is it okay with the terrain access patterns?

Reply:
a 10km radius is 2.5Mb of data, so perfectly fine on the W25NXX chips

Reply:
The problem is that the "Local Area" would need to be flashed in the firmware right? I usually load about 200km2 of STRM30 data around Edmonton when I load terrain on a new AP which works out to about 4G.
How would I do that?